messagequeue/messagequeue.go

```diff
@@ -30,7 +38,8 @@ type MessageQueue struct {
 	done chan struct{}
 }
 
-func New(p peer.ID, network bsnet.BitSwapNetwork) *MessageQueue {
+// New creats a new MessageQueues
```

Comment:
nitpick FYI: comments should end with a fullstop, see [here](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences), please don't feel it necessary to update all the comments but moving forward it would be great to standardize on this.

Reply:
yea I can commit to this one going forward. :)

Reply:
went ahead and implemented this across bitswap as best I could, but put this in a seperate commit (including spelling change above) to be merged right after, so as to avoid making review harder. https://github.com/ipfs/go-bitswap/pull/31